cli/tests/integrations/test_marathon.py

```diff
@@ -144,16 +144,24 @@ def test_show_relative_app_version():
 
 
 def test_show_missing_relative_app_version():
+    app_id = 'zero-instance-app'
```

Comment:
this could be a global `ZERO_INSTANCE_APP_ID` since we use it all over this file